core/zoom_controls.js

```diff
@@ -260,16 +253,28 @@ Blockly.ZoomControls.prototype.createZoomInSvg_ = function(rnd) {
       },
       this.zoomInGroup_);
   zoominSvg.setAttributeNS(Blockly.utils.dom.XLINK_NS, 'xlink:href',
-      ws.options.pathToMedia + Blockly.SPRITE.url);
+      this.workspace_.options.pathToMedia + Blockly.SPRITE.url);
 
   // Attach listener.
-  Blockly.bindEventWithChecks_(zoominSvg, 'mousedown', null, function(e) {
-    ws.markFocused();
-    ws.zoomCenter(1);
-    Blockly.Touch.clearTouchIdentifier();  // Don't block future drags.
-    e.stopPropagation();  // Don't start a workspace scroll.
-    e.preventDefault();  // Stop double-clicking from selecting text.
-  });
+  Blockly.bindEventWithChecks_(
+      zoominSvg, 'mousedown', null, this.zoom_.bind(this, 1));
+};
+
+/**
+ * Zooms in or out of the workspace.
```

Comment:
I think this is too generic of a description for this method. This isn't a method for zooming in / out, it's a method that handles a mouse event for both zoom in and out buttons, perhaps something like: 

"Handles a mouse-down on the Zoom in or Zoom out buttons on the workspace."